modulemd/meson.build

```diff
@@ -143,6 +147,7 @@ test_v2_priv_hdrs = files(
 )
 
 test_v2_python_scripts = files(
+    'v2/tests/ModulemdTests/translationentry.py',
```

Comment:
Please include `base.py` here as well. This is the list of files that gets run through autopep8 for formatting.